test/geometry/test_ransac.py

```diff
@@ -0,0 +1,108 @@
+import random
+
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils
+from kornia.geometry import RANSAC
+from kornia.testing import assert_close
+
+
+class TestRANSACHomography:
```

Comment:
add a value tests as in the fundamental. Check if jit, onnx, (gradcheck?) work here

Reply:
Why? We already have numeric test in dirty_points

Reply:
Will add jit thing